csharp/src/ProtocolBuffers/ByteString.cs

```diff
@@ -139,7 +139,7 @@ public static ByteString FromBase64(string bytes)
         /// are copied, so further modifications to the array will not
         /// be reflected in the returned ByteString.
         /// </summary>
-        public static ByteString CopyFrom(byte[] bytes)
+        public static ByteString CopyFrom(params byte[] bytes)

```

Comment:
Given this is part of public API and some users will be surprised by having to use ByteString instead of just byte[], let's not make the thing even more confusing to them and let's add an example of usage of this method to the summary. 
the semantics of CopyFrom(1,2,3) call might not be obvious to everyone, so and example might be helpful for them.